pkg/lib/bundle/supported_resources.go

```diff
@@ -16,7 +16,7 @@ const (
 	PodDisruptionBudgetKind   = "PodDisruptionBudget"
 	PriorityClassKind         = "PriorityClass"
 	VerticalPodAutoscalerKind = "VerticalPodAutoscaler"
-	ConsoleYamlSampleKind     = "ConsoleYamlSample"
+	ConsoleYAMLSampleKind     = "ConsoleYAMLSampleKind"
```

Comment:
Is the `Kind` part expected on the value side?

Reply:
No! Thank you. Copy paste bug from the repeated tests. I've been upgrading my go to be able to run the e2e locally.